test.js

```diff
@@ -434,3 +434,33 @@ test('`clearInvalidConfig` option - valid data', t => {
 	conf.set('foo', 'bar');
 	t.deepEqual(conf.store, {foo: 'bar'});
 });
+
+test('schema - should be an object', t => {
+	const schema = 'object';
+	t.throws(() => new Conf({cwd: tempy.directory(), schema}), 'Schema option must be an object.');
+});
+
+test('schema - invalid set', t => {
+	const schema = {
+		foo: {
+			type: 'string'
+		}
+	};
+	const conf = new Conf({cwd: tempy.directory(), schema});
+	conf.set('foo', fixture);
+	t.is(conf.get('foo'), fixture);
+	t.throws(() => conf.set('foo', 1), 'Config is not valid according to schema: .foo should be string;');
```

Comment:
The error should be:

```
Config schema violation: `foo` should be string
```

Not that there should be not `.` in front of `foo`. And it should be wrapped in backticks, and no `;` at the end.